test/ui-testing/publishButton.test.js

```diff
@@ -7,8 +7,16 @@ beforeAll(async () => {
 
 describe('Publish button', () => {
   
-  test('something we are testing described here', () => {
-
+  test('Publish button is enabled', () => {
+    // Check initially that Publish button is disabled.    
+    expect(await page.evaluate(() => document.querySelector('.ple-textarea').disabled).toBe(true);
```

Comment:
What's odd is that I thought we use the /Bootstrap class/ `disabled` to set the disabled state on this. I don't know how that interacts with the HTML attribute -- the difference is:

`<i class="disabled"></i>` (Bootstrap CSS)
`<i disabled></i>` (HTML)

So we may need to disambiguate how this is working, possibly by manually testing the jquery `$('.ple-publish').prop('disabled')` vs. just `el.disabled`.